### PR TITLE
feat(html/minifier): compress default values of attributes

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Compress default values.

There are two limitation (so they were not added in default):
1. Some attributes have `default` value in spec, but some browsers don't respect spec and use own values (Safari like it), so they were not added
2. Legacy attributes (like `align`/etc), the same problems for different browsers

In any case, there can be many them, I tried to collect the most popular

**BREAKING CHANGE:**

No

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_html_minifier/src/lib.rs [hunk 1]
```diff
@@ -70,6 +150,19 @@ impl VisitMut for Minifier {
         n.visit_mut_children_with(self);
 
         n.children.retain(|child| !matches!(child, Child::Comment(comment) if !self.is_conditional_comment(&comment.data)));
+        n.attributes.retain(|attribute| match &*attribute.name {
+            _ if attribute.value.is_some()
```
Comment (kdy1) [inline crates/swc_html_minifier/src/lib.rs:154]: I think we don't need if guard here

Comment (kdy1) [reply crates/swc_html_minifier/src/lib.rs:154]: We can directly return the negated boolean for if-guard

**Review by kdy1 — CHANGES_REQUESTED** [review]

**Comment by alexander-akait** [comment]
@kdy1 Rebased and refactored

**Review by kdy1 — APPROVED** [review]
Thanks!

---

swc-bump:
 - swc_html_minifier

**Comment by alexander-akait** [comment]
@kdy1 Rebased and tests fixed, ready to merge

**Review by kdy1 — APPROVED** [review]

**Review by kdy1 — COMMENTED** [review]
reset ci

**Pull request merged**